addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -459,3 +459,38 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
         }
     }
 }
+
+/**********************************************************
+            Image (Zoom widget) Specific Style
+ **********************************************************/
+
+.o_survey_image_zoom_lightbox {
```

Comment:
Let's try to stick to a single "namespace", like `o_survey_choice_image` ?

Reply:
o_survey_choice_image_modal in this case then ?